### DISCUSSION TITLE
M16 4060

### DISCUSSION THREAD

**Discussion by yuanwei9808** [seerge/g-helper#1310] [Ideas]
Can the PL1 and PL2 of the M16 4060 be adjusted separately?

**Comment by seerge** [comment]
@yuanwei9808 no, I have merged 2 settings into one for simplicity in all cases. What's a real practical application for that ? :)

**Comment by IceStormNG** [comment]
Not with GHelper and most people want Pl1 = PL2 as it is more predictable and sustained. If you want to edit them independently, use Throttlestop. You can use that without issues alongside GHelper.
